### ISSUE TITLE
Deprecate ember-bootstrap blueprints

### ISSUE THREAD

**Issue by SanderKnauff** [ember-bootstrap/ember-bootstrap#2090]
Since V2 addons do not support blueprints, the ember-bootstrap blueprint should be deprecated before the conversion.
The documentation contains instructions that use the blueprints, so these need to be updated to instead describe the manual steps required for configuring the addon.

**Comment by jelhan** [comment]
How would deprecating the blueprints look like? If consumer installs `ember-bootstrap` using `ember install`, the blueprints are executed automatically. I fear we can only remove them without deprecating upfront.

Also deprecating may not be worse it. Typically the blueprints are only run once on installation. So likely a consumer sees the deprecation warning only once (if ever).

I wonder if we should consider dropping the blueprints a breaking change. It would not affect any app, which are using `ember-bootstrap` already. It would only affect projects who have setup some automation for generating new Ember apps, which rely on our blueprints being executed. But I feel it is unlikely that this exists in practice. And if it exists, I wouldn't consider it as being official support.

Long story short: I would be fine with removing the blueprints in the next minor release.

**Comment by SanderKnauff** [comment]
The documentation does mention an automated way to change the project configuration from Bs4/Bs5 using `ember-generate`. [1]
I'm not sure how actively used it is, but that would be a possible place to add a deprecation notice.
That said, I don't mind removing it if that is preferred. That also means that we can remove the node tests that are currently living in the addon project.

[1]  https://www.ember-bootstrap.com/getting-started/setup
